01_dask.delayed.ipynb

```diff
@@ -70,12 +70,9 @@
    "cell_type": "markdown",
```

Comment:
<p>I would remove the "returning a 'Delayed' object" part. I don't think it adds much at this point.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-tutorial/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I think it is a good suggestion to move this text below as you say in your next comment.